airbyte-webapp/src/components/base/SecretTextArea/SecretTextArea.tsx

```diff
@@ -0,0 +1,93 @@
+import { faEye } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+import { useMemo, useRef } from "react";
+import { FormattedMessage } from "react-intl";
+import { useToggle, useUpdateEffect } from "react-use";
+
+import { TextInputContainer, TextInputContainerProps } from "../TextInputContainer";
+import styles from "./SecretTextArea.module.scss";
+
+interface SecretTextAreaProps
+  extends Omit<TextInputContainerProps, "onFocus" | "onBlur">,
+    React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+export const SecretTextArea: React.VFC<SecretTextAreaProps> = ({
```

Comment:
```suggestion
export const SecretTextArea: React.FC<SecretTextAreaProps> = ({
```

So we're not introducing new deprecated API with the React 8 upgrade please (also see [this Slack message](https://airbytehq-team.slack.com/archives/C039SG3DQ8K/p1663178974178799)).